src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParser.g4

```diff
@@ -908,7 +910,9 @@ javadocInlineTag:
             | LINKPLAIN_LITERAL (WS | NEWLINE | LEADING_ASTERISK)* reference description?
             | LITERAL_LITERAL (WS | NEWLINE | LEADING_ASTERISK | text)*
             | VALUE_LITERAL (WS | NEWLINE | LEADING_ASTERISK)* reference?
-            | CUSTOM_NAME (WS | NEWLINE | LEADING_ASTERISK)* description?
+            | CUSTOM_NAME (WS | NEWLINE | LEADING_ASTERISK)*
+              ({!isNextJavadocTag()}? LEADING_ASTERISK | htmlComment
+              | CDATA | NEWLINE | text | javadocInlineTag | htmlElement)*
```

Comment:
I think when someone adds a custom inline javadoc tag - the aim is to add a custom processing of the text inside this tag and since we cannot know what kind of processing it is we should treat it as just text.